.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- 
+Please ensure that you complete the following mandatory sections:
+
+- Issue*
+- Overview
+- Reason*
+- Work carried out
+- Confirmations
+
+* Only mandatory if working from a issue
+ -->
+
+## 🧰 Issue
+<!-- [The issue the work was done for as an issue reference (e.g. `closes #1`)] -->
+
+## 🚀 Overview: 
+<!-- [A summary of what you did in no more than one paragraph] -->
+
+## 🔗 Link to preview (or screenshot, if relevant)
+<!-- [If you're on a project which auto-deploys branches, link to the branches preview link here] -->
+
```

Comment:
_Maybe_ we'll end up with a deployment pipeline where we have a _playable_ version of PRs, maybe utilising `binder`.  This would both reduce the "cost" in reviewing a PR, and potentially allow analysts to contribute to review.